projects/backend/index.ts

```diff
@@ -49,6 +50,15 @@ app.get('/', (req, res) => {
     res.send(JSON.stringify({ client: '🦆' }))
 })
 
+const endpoints = listEndpoints(app)
+
+const docsPath = '/docs'
+
+app.get(docsPath, (req, res) => {
```

Comment:
I'd be tempted to use the route path here, to be really explicit for newcomers.

Reply:
yeah make sense

Reply:
done

Reply:
root?

Reply:
Lol, yeah, root. Oops!